crates/revm/src/processor.rs

```diff
@@ -248,30 +257,40 @@ where
     ) -> Result<ResultAndState, BlockExecutionError> {
         // Fill revm structure.
         #[cfg(not(feature = "optimism"))]
-        fill_tx_env(&mut self.evm.env.tx, transaction, sender);
+        fill_tx_env(&mut self.evm.context.evm.env.tx, transaction, sender);
```

Comment:
there should be getters for this on evm imo

Reply:
Added cfg,tx,block handlers and their `_mut` counterparts